src/miner_keys.erl

```diff
@@ -20,6 +20,8 @@
 -spec keys({file, BaseDir::string()} |
            {ecc, proplists:proplist()}) -> key_info().
 keys({file, BaseDir}) ->
+    BootDir = "/mnt/uboot",
+    OnboardingKey = filename:join([BootDir, "onboarding_key"]),
```

Comment:
So two options:

1. Rename to OnboardingDir and get it from sys.config instead of hardcoding here
2. Mount the folder in BaseDir somewhere and have OnboardingKey just offset from BaseDir

Reply:
I am going to go with option 1.